apis/task.py

```diff
@@ -223,9 +223,9 @@ def run_model(self) -> DAGNode:
       workload_id = xpk.generate_workload_id(self.task_test_config.benchmark_id)
       run_workload = xpk.run_workload(
           task_id="run_workload",
-          project_id=self.task_gcp_config.project_name,
+          cluster_project=self.task_gcp_config.cluster_project,
```

Comment:
Does GCPConfig.cluster_project get populated if it's omitted on construction, or could this still carry the default `None` value?

Reply:
Thanks Jon! Good question. I didn't put a default value for cluster_project. Actually, I think it's better to add composer_project to indicate the project that hosts env info for post_process step (like [this](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/48/files#diff-82effdf45298e1c77fe8b16705253cd0896789bcf2074caf4726fed50af68226)). I will merge that change into this one.